src/components/InlineSystemMessage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import Text from './Text';
+import * as Expensicons from './Icon/Expensicons';
+import colors from '../styles/colors';
+import Icon from './Icon';
+
+const propTypes = {
+    /** Error to display */
+    message: PropTypes.string.isRequired,
+};
+
+const InlineSystemMessage = (props) => {
+    if (props.message.length() === 0) {
```

Comment:
I think this should be `props.message.length`, as I'm getting a length is not a function error

Reply:
Ah, totally right. My bad, as I added this at the end 😬